source/opt/module.h

```diff
@@ -51,7 +51,7 @@ class Module {
   // Sets the header to the given |header|.
   void SetHeader(const ModuleHeader& header) { header_ = header; }
   // Sets the Id bound.
-  void SetIdBound(uint32_t bound) { header_.bound = bound; }
+  inline void SetIdBound(uint32_t bound) { header_.bound = bound; }
```

Comment:
When the body is present at the declaration of a method in a class, then it's already implicitly inline, IIRC.